hr_expense_invoice/models/hr_expense.py

```diff
@@ -153,27 +128,26 @@ def _onchange_invoice_id(self):
         """
         if self.invoice_id:
             self.quantity = 1
-            self.reference = self.invoice_id.name
```

Comment:
Maybe we should add the vendor bill number in the expense name?

Reply:
Done but since both models, `hr.expense` and `account.move` have smart buttons to go from one to another and since in v17 the filtering is more robust. I think is not necessary to have it. Maybe something like:
```
self.name = self.name or self.invoice_id.name
```
What do you think?